recipes/jquery/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jquery_static" %}
+{% set version = "3.2.1" %}
+{% set sha256 = "31d8c2e22dd48a7973fe3617cc656fe6b64d2250d07a258cd8ab30ead031c24a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jquery/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  run:
+    
+test:
+  commands:
+    - npm list -g jquery
```

Comment:
You'll need `nodejs` in `test/requires` to run this, since it isn't a run requirement.